lib/abn/entity.rb

```diff
@@ -41,9 +41,9 @@ def initialize(attributes = {})
     #
     # @return [String] business name
     def best_name
-      trading_name ||
+      main_name ||
```

Comment:
Shouldn't trading_name be first preference? If a business has a trading_name it's because they want to use it as their public facing name.

Reply:
tsk tsk chester

Reply:
LOL. your terrible naming was messing with me bro.

Reply:
That's how the API returns it -_- shhhh

Reply:
forgiven. we still miss u <3

Reply:
miss all you guys too <3